.github/workflows/wrapper.yml

```diff
@@ -8,4 +8,8 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v4
-      - uses: gradle/wrapper-validation-action@v3
+      - name: Retry Gradle Wrapper Validation
```

Comment:
Please open a separate issue or PR for this change. It looks like this action has been [archived](https://github.com/gradle/wrapper-validation-action) and superseded by a different action, so perhaps we should first update to that.